_partials/_devops-mcp-commands.md

```diff
@@ -17,6 +17,16 @@ $MCP_LONG exposes the following MCP tools to your AI Assistant:
+|                          | `fork_strategy`     | ✓        | Fork strategy: <ul><li>`NOW`: Fork at current database state</li><li>`LAST_SNAPSHOT`: Fork at last existing snapshot (faster)</li><li>`PITR`: Point-in-time recovery to specific timestamp (requires `target_time`)</li></ul>                                                            |
+|                          | `target_time`       | -        | Target timestamp for point-in-time recovery (RFC3339 format, e.g., `2025-01-15T10:30:00Z`). Only used when `fork_strategy` is `PITR`.                                                                                                                                                   |
+|                          | `name`              | -        | Set the human-readable name for the forked $SERVICE_SHORT. Defaults to `{source-service-name}-fork`.                                                                                                                                                                                     |
+|                          | `cpu_memory`        | -        | CPU and memory allocation combination. Inherits from source $SERVICE_SHORT if not specified. <br /> Available configurations are: <ul><li>shared/shared</li><li>0.5 CPU/2 GB</li><li>1 CPU/4 GB</li><li>2 CPU/8 GB</li><li>4 CPU/16 GB</li><li>8 CPU/32 GB</li><li>16 CPU/64 GB</li><li>32 CPU/128 GB</li></ul> |
+|                          | `wait`              | -        | Set to `true` to wait for forked $SERVICE_SHORT to be fully ready before returning. Default: `false`.                                                                                                                                                                                    |
+|                          | `timeout_minutes`   | -        | Set the timeout in minutes to wait for forked $SERVICE_SHORT to be ready. Only used when `wait=true`. Default: 30 minutes                                                                                                                                                               |
+|                          | `set_default`       | -        | By default, the forked $SERVICE_SHORT is set as the default for following commands in $CLI_SHORT. Set to `false` to keep the previous $SERVICE_SHORT as the default.                                                                                                                     |
+|                          | `with_password`     | -        | Set to `true` to include the password for the forked $SERVICE_SHORT in response and connection string. <br/> **WARNING**: never set to `true` unless user explicitly requests the password.                                                                                              |
```

Comment:
This is just adding documentation for the new `service_fork` MCP tool that was added recently.

Reply:
Thanks